testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketSslClientRenegotiateTest.java

```diff
@@ -50,16 +49,20 @@
 import java.util.concurrent.Executor;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 
 import javax.net.ssl.SSLHandshakeException;
 
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
-@RunWith(Parameterized.class)
 public class SocketSslClientRenegotiateTest extends AbstractSocketTest {
+    // HACK: JUnit 5 ParameterizedTest does not allow a data source with zero arguments
+    private static final SslContext JUNIT_5_PARAMETERIZED_HACK = null;
```

Comment:
I am not sure if there is a less-hack way to accomplish this - please let me know if you have another solution.

Reply:
@chrisvest any ideas ?

Reply:
@kashike @normanmaurer Use a `@DisabledIf` annotation to point to a method that will check if OpenSSL is available or not.